linter/allowlists/default.txt

```diff
@@ -242,6 +244,7 @@ sql
 start
 Start
 Stardate
+Sticky
```

Comment:
This should go to the German or English allowlist.

Reply:
"Sticky" is in the name of the extension, which is the same in german and english. So it belongs to the default.txt?

Reply:
"Sticky", is an English adjective. But, well... :)